### DISCUSSION TITLE
sign in

### DISCUSSION THREAD

**Discussion by yegor25** [vercel/community#4028] [Account Issue] [answered]
### Question

hello. I have sign up in your platform, got message on gmail, clicked button verify, but I can't login in your platform.
My email is egorlesnicij86@gmail.com

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
